Form/ChoiceList/ModelChoiceLoader.php

```diff
@@ -26,6 +26,8 @@
  */
 class ModelChoiceLoader implements ChoiceLoaderInterface
 {
+    public $identifier;
```

Comment:
I don't know what this is supposed to be, and it's not meant to stay public forever BTW.